reg/types.go

```diff
@@ -145,6 +145,7 @@ type Info uint8
 const (
 	None       Info = 0
 	Restricted Info = 1 << iota
+	BasePointer
```

Comment:
Not sure if this is the right property name. An alternative is to call it something like `AutoSaveRestore`. The distinction is naming it for what it is vs. what the Go assembler does with it.